scripts/build-container.sh

```diff
@@ -10,4 +10,4 @@ podman build --no-cache --build-arg=VERSION=${OLS_VERSION} -t ols:latest -f Cont
 # podman tag ols:latest quay.io/<org>/ols:$OLS_VERSION
 
 # To test-run for local development
-# podman run --rm -ti -p 8080:8080 localhost/ols:latest
\ No newline at end of file
+# podman run --rm -ti --env-file=.env -p 8080:8080 localhost/ols:latest
```

Comment:
you want to encode this as a new Makefile target, like `run-image` ?

Reply:
No particular preference. This is the exact command I use to test this during dev